src/gui/sdl_mapper.cpp

```diff
@@ -642,6 +642,15 @@ class CStickBindGroup : public CBindGroup {
 			LOG_MSG("SDL: Can't detect axes; %s", SDL_GetError());
 		axes = clamp(sdl_axes, 0, MAXAXIS);
 
+		// Axis position is reported as a signed 16 bit integer
+		// On a typical gamepad, analogue sticks have a resting position of 0
+		// Trigger buttons have a resting position close to -32000
+		// Probe the axes and if we have one with a very low value, we're probably dealing with an analogue trigger
+		// Hope the user isn't mashing their controller while we're initalizing :)
```

Comment:
Yeah, this is as good as it gets until do the proper [GameControllerDB](https://github.com/mdqinc/SDL_GameControllerDB) integration.

Nice comments!